dbconnection.js

```diff
@@ -0,0 +1,15 @@
+const configuration = {
+    user: 'sa',
```

Comment:
Let's store these sensitive values into an `.env` file for now, https://www.freecodecamp.org/news/how-to-use-node-environment-variables-with-a-dotenv-file-for-node-js-and-npm/